### ISSUE TITLE
The new too-narrow-bins warning is wrong

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#194]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/194) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

When KAlpha or other line fits are done with pulse height units, which might be a factor of several smaller than energy units, it can too easily trigger Paul’s new warning \(@{557058:6b1a4860-1eb7-4caa-9340-5e3c97a89faf} \) about bins that are too narrow. I think either the dPH/dE factor is being ignored, or its inverse is being used.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Example code that triggers what I’m talking about:

```python
        line = mass.calibration.fluorescence_lines.MnKAlpha
        model = line.model()
        N = 10000
        np.random.seed(238)
        energies = line.rvs(size=N, instrument_gaussian_fwhm=4.0)  # draw from the distribution

        SCALE = 3.0
        sim, bin_edges = np.histogram(energies*SCALE, 60, [5865*SCALE, 5925*SCALE])
        binsize = bin_edges[1] - bin_edges[0]
        bctr = bin_edges[:-1] + 0.5*binsize
        params = model.guess(sim, bin_centers=bctr)
        params["peak_ph"].set(value=5899*SCALE)
        result = model.fit(sim, params, bin_centers=bctr)
        print(result.fit_report())
```

This generates 10000 random MnKAlpha energies. It then bins 3x the energies into a histogram and tries to fit, but it generates the warning. It should not. I’ll turn this into a test and fix it.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fix fitting with dPH/dE >> 1

Fixes #194. I think Paul or Galen misunderstood that the `fwhm` parameter is always in energy units, and does not need to be divided by `dph_de` to get energy units.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
